src/multi_dim_support/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
So the stamped out Chapel code will be placed here?

Reply:
Yep, it goes in a single source file in this directory that gets compiled with everything else.